src/styles/blocks/catalog.scss

```diff
@@ -1,25 +1,34 @@
 .catalog {
   display: flex;
   flex-wrap: wrap;
-
   max-width: 944px;
   margin: 0 auto;
   padding: 50px 0;
 
   &__card {
+    $card-width: 200px;
+    $card-height: 300px;
+    $card-margin: 48px;
+    $hover-duration: 300ms;
     display: flex;
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
-
-    width: 200px;
-    height: 300px;
+    width: $card-width;
+    height: $card-height;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    margin-bottom: $card-margin;
+    transition-duration: $hover-duration;
```

Comment:
Add 'transition-property' to specify which properties should have the transition effect